global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.100",
-    "rollForward": "latestFeature"
+    "version": "6.0.202",
```

Comment:
Why did we need to change the `rollForward` behavior here? Is there something about `6.0.300` that we don't want?

Reply:
latestMajor means the build server will use .NET 7 SDK which install-dotnet.sh downloads and installs. Its required because the new example uses .NET 7.

I didn't update the version number here beyond 6 because this global.json is used by all source, including some code that is tested in containers. Those containers don't support the .NET 7 version we need.

This allows everything to work with a minimal amount of hackery.